packages/dd-trace/src/appsec/rule_manager.js

```diff
@@ -69,6 +69,23 @@ function copyRulesData (rulesData) {
   }
   return copy
 }
+
+function getRulesOverrideData (asmData) {
+  return asmData.rules_override && Array.isArray(asmData.rules_override) &&
+    asmData.rules_override
+      .filter(ruleOverride => ruleOverride.hasOwnProperty('enabled'))
+      .map(ruleOverride => ({ id: ruleOverride.id, enabled: ruleOverride.enabled }))
+}
+
+function toggleRules (action, asmData) {
+  const rulesOverrideData = getRulesOverrideData(asmData)
```

Comment:
What happens when the rule is disabled and enabled again from backend? we receive the rule with `enabled: true` again?